java/src/org/openqa/selenium/remote/RemoteWebDriverBuilder.java

```diff
@@ -438,7 +452,13 @@ private DriverService startDriverServiceIfNecessary() {
 
   private CommandExecutor createExecutor(HttpHandler handler, ProtocolHandshake.Result result) {
     Dialect dialect = result.getDialect();
-    Function<Command, HttpRequest> commandEncoder = dialect.getCommandCodec()::encode;
+    CommandCodec<HttpRequest> commandCodec = dialect.getCommandCodec();
+    for (Map.Entry<String, CommandInfo> entry : additionalCommands.entrySet()) {
+      commandCodec.defineCommand(
+          entry.getKey(), entry.getValue().getMethod(), entry.getValue().getUrl());
+    }
```

Comment:
This is nit. The change does not need to be incorporate. Just a suggestion, this can use forEach: 
```
 additionalCommands.forEach(
        (name, commandInfo) ->
            commandCodec.defineCommand(name, commandInfo.getMethod(), commandInfo.getUrl()));
```

Reply:
I copied straight from HttpCommandExecutor class, but probably it does that because it is using two steps, I think to test for nulls in entry set? At any rate, an easy change to update.

Reply:
I think both do the same thing.